server/lib/queries.js

```diff
@@ -667,6 +670,51 @@ const getTotalNumberOfDonors = () => {
   .then(res => parseInt(res[0].count));
 }
 
+const getCollectivesWithMinBackers = memoize(async ({ backerCount = 0, orderBy = 'createdAt', orderDirection = 'ASC', limit = 0, offset = 0 }) => {
+  const params = {
+    model: models.Collective,
+  };
+
+  const sql = (fields) => `
+    with "actives" as (
+      SELECT c.id
+      FROM "Collectives" c
```

Comment:
Better to add the conditions on `Collectives` here to reduce the selectivity before doing the `JOIN` (i.e. `WHERE c."isActive" IS TRUE and c."deletedAt" IS NULL`